jquery.inputMachinator.js

```diff
@@ -51,21 +51,32 @@ OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
         var $selects = $();
 
         // Check which controls machinator should run on
-        if (options.checkbox)
+        if (options.checkbox) {
+            
```

Comment:
Why are there empty lines within the braces?

Reply:
My mistake, I forgot to remove.